eraser-map.yaml

```diff
@@ -1325,7 +1321,7 @@ layers:
                             width: [[9, 1.0px], [10, 1.0px], [11, 1.2px], [12, 1.25px], [13, 1.5px], [14, 1.6px], [15, 0.15px], [16, 5.25px], [17, 12m]]
                             outline:
                                 color: [[13, [0.870,0.870,0.870]], [15, [0.976,0.612,0.573]]]
-                                width: [[9, 0.0px], [12, 0px], [15, 2px], [16, 2px], [17, 3px]]
+                                width: [[13, 0], [15, 2px], [16, 2px], [17, 3px]]
```

Comment:
Yeah I was sloppy about this, I originally was going to update all the `0px` to be `0`, because there should be an advantage to not having to do unit conversion from pixels to meters, but then didn't want to expand scope of the PR. This may be silly though, Tangram itself can just remove the units for a zero value. (There's still probably a slight parsing advantage because it only has to look for units if the value is a JS string, if it's a pure JS number it already knows there's not unit.) I can benchmark to see how meaningful (if at all) the difference is.